cli/config/configuration-qemu-virtiofs.toml.in

```diff
@@ -16,6 +16,19 @@ kernel = "@KERNELVIRTIOFSPATH@"
 image = "@IMAGEPATH@"
 machine_type = "@MACHINETYPE@"
 
+# List of valid annotation names for the hypervisor
+# Each member of the list is a regular expression, which is the base name
+# of the annotation, e.g. "path" for io.katacontainers.config.hypervisor.path"
+# The default if not set is empty (all annotations rejected.)
+# Your distribution recommends: @DEFENABLEANNOTATIONS@
```

Comment:
q. why the comment here is different to the rest of `.toml.in` files?

```toml
# List of valid annotation names for the hypervisor
# Each member of the list is a regular expression, which is the base name
# of the annotation, e.g. "path" for io.katacontainers.config.hypervisor.path"
````

Reply:
Interesting. I thought I had unified all the docs. Good catch, I will align all the others to the on for virtiofs (which is more verbose, preferred).

Good point about the docs as well. I will submit a different PR for that.

Reply:
Ah, I take that back. There are two distinct comments, and they are consistent across files.

The `enable_annotations` configuration checks the _names_ of the annotations you will accept. For example, you may want to accept only `kernel_params` annotations, but reject an annotation that modifies the hypervisor.

A second level of check is that if you accept the hypervisor annotation, you can still restrict which values are accepted.

Reply:
Any suggestion for a clearer wording is welcome, BTW.

Reply:
I like this comment, just make it consistent across `toml.in` files (all file should have the same comments)

Reply:
> I like this comment, just make it consistent across `toml.in` files (all file should have the same comments)

Sorry, I somehow had missed the inconsistency, despite checking. I saw it now.

Reply:
I will make it a separate commit so that I can apply on 2.0 where the changes were already merged.